doc/release/technotes/README.auxIO.rst

```diff
@@ -12,8 +14,13 @@ systems simultaneously.
 Dependencies
 ------------
 
-For each AIO system described here, there is a corresponding README.<system> in
-this directory detailing how to install and use them.
+Each AIO system is typically described in the support module for it.
+See:
+
+ * :mod:`HDFS`
```

Comment:
That makes a cross-reference to the Chapel module called HDFS. It works for me (in my rendered HTML).